diff --git a/electron/services/preferences-manager.ts b/electron/services/preferences-manager.ts
index acd4c42..6b16ec1 100644
--- a/electron/services/preferences-manager.ts
+++ b/electron/services/preferences-manager.ts
@@ -1,4 +1,5 @@
-import { app, shell, BrowserWindow, ipcMain, IpcMainInvokeEvent } from 'electron'
+import type { IpcMainInvokeEvent } from 'electron';
+import { app, shell, BrowserWindow, ipcMain } from 'electron'
 import { join } from 'path'
 import { exec } from 'child_process'
 import { cache, CacheAttribute } from '../config/index'